dev/signer/signer1.tf

```diff
@@ -0,0 +1,11 @@
+resource "helm_release" "signer" {
```

Comment:
Should probably keep all of this in `signer/main.tf`. The size of `signer/main.tf` with this content isn't so big (yet) that it would warrant a split into a new file.

Reply:
changed in [c575a02](https://github.com/GaloyMoney/charts/pull/1581/commits/c575a02f6365743ae3a32777ec19faac4e11c676)